packages/mux-player/src/template.ts

```diff
@@ -41,10 +36,11 @@ const getHotKeys = (props: MuxTemplateProps) => {
 export const content = (props: MuxTemplateProps) => html`
   <media-theme
     template="${props.themeTemplate ?? muxTemplate.content.children[0]}"
-    stream-type="${isLiveOrDVR(props) ? 'live' : 'on-demand'}"
-    target-live-window="${isDVR(props) ? 1 : false}"
+    default-stream-type="${props.defaultStreamType ?? false}"
+    class="${props.secondaryColor ? 'two-tone' : false}"
     hotkeys="${getHotKeys(props) || false}"
     nohotkeys="${props.noHotKeys || !props.hasSrc || props.isDialogOpen || false}"
+    noautoseektolive="${props.targetLiveWindow !== 0}"
```

Comment:
disables "auto seek to live" on play requests for non "standard live" cases. This fixes an issue with pause+play for "DVR" but does not fully resolve "simulated live" cases (where the content is "on demand" but we want to "treat it as live," which were likewise already broken).